src/consumer.ts

```diff
@@ -68,6 +68,14 @@ function isConnectionError(err: Error): boolean {
   return false;
 }
 
+function isAbortedError(err: Error): boolean {
+  if (err instanceof SQSError) {
+    return err.statusCode === 200 || err.code === 'RequestAbortedError';
```

Comment:
Not sure we want to consider everything with a status code 200 as an abort error...